projects/SCION/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "SCION is a clean-slate Next-Generation Internet (NGI) architecture which offers a.o. multi-path and path-awareness capabilities by design.";
+    subgrants = [
+      "SCION-proxy"
+      "SCION-router-codealignment"
+      "Verified-SCION-router"
+      "SCION-Rains"
+      "SCION-Swarm"
+      "SCION-IPFS"
+      "SCION-1M"
+    ];
+  };
+
+  nixos.modules.programs = {
+    scion = {
+      name = "scion";
+    # module = ./module.nix;
```

We have some packages in `projects-old/SCION-1M` which we can put in the `nixos.modules.programs.scion` module.

```suggestion
    module = ./programs/module.nix;
```

Inside `./programs/module.nix` (which you'd need to create):

```nix
{
  config,
  lib,
  pkgs,
  ...
}:
let
  cfg = config.programs.scion;
in
{
  options.programs.scion = {
    enable = lib.mkEnableOption "SCION";
  };

  config = lib.mkIf cfg.enable {
    environment.systemPackages = with pkgs; [
      scion
      scion-apps
      scion-bootstrapper
      ioq3-scion
      pan-bindings
    ];
  };
}
```